linenoise/sbom_linenoise.yml

```diff
@@ -0,0 +1,3 @@
+name: sbom_linenoise
+description: Line editing C library
+url: https://github.com/antirez/linenoise
```

Comment:
The library is not maintained upstream anymore, i think the URL should point to idf-extra-components instead.

Please also check the other fields:

```yml
name: linenoise
version: 2.0.0
cpe: cpe:2.3:a:espressif:linenoise:{}:*:*:*:*:*:*:*
supplier: 'Organization: Espressif Systems'
```